schemas/exchange/min-meta/json-min-db-pdbx_core-col-pdbx_core_entry.json

```diff
@@ -10289,6 +10290,944 @@
+                     }
+                  ]
+               },
+               "percent_free_reflections": {
+                  "type": "number",
+                  "description": "A percentage, Normally percent proportion of the total number. Between 0% and 100%.",
+                  "rcsb_description": [
+                     {
+                        "text": "A percentage, Normally percent proportion of the total number. Between 0% and 100%.",
+                        "context": "dictionary"
+                     }
+                  ]
+               },
+               "servalcat_version": {
+                  "type": "string",
+                  "description": "The version of Servalcat program used in the analysis.",
+                  "rcsb_description": [
+                     {
+                        "text": "The version of Servalcat program used in the analysis.",
+                        "context": "dictionary"
+                     }
+                  ]
+               },
+               "trans_NCS_details": {
+                  "type": "string",
+                  "examples": [
+                     "The largest off-origin peak in the Patterson function is 8.82% of the height of the origin peak. No significant pseudotranslation is detected."
+                  ],
+                  "description": "A sentence giving the result of Xtriage's analysis on translational NCS.\nX-ray entry specific, obtained from the Xtriage program.",
+                  "rcsb_description": [
+                     {
+                        "text": "A sentence giving the result of Xtriage's analysis on translational NCS.\nX-ray entry specific, obtained from the Xtriage program.",
+                        "context": "dictionary"
+                     }
+                  ]
+               },
+               "twin_fraction": {
+                  "type": "string",
+                  "examples": [
+                     "h,h-k,h-l:0.477;-h,-h+k,-l:0.020;-h,-k,-h+l:0.017"
+                  ],
+                  "description": "Estimated twinning fraction for operators as identified by Xtriage. A semicolon separated\nlist of operators with fractions is givens \nX-ray entry specific, obtained from the Xtriage program.",
+                  "rcsb_description": [
+                     {
+                        "text": "Estimated twinning fraction for operators as identified by Xtriage. A semicolon separated\nlist of operators with fractions is givens \nX-ray entry specific, obtained from the Xtriage program.",
+                        "context": "dictionary"
+                     }
+                  ]
+               }
+            },
+            "additionalProperties": false
+         },
+         "minItems": 1,
+         "uniqueItems": true
+      },
+      "pdbx_vrpt_summary_em": {
+         "type": "array",
+         "items": {
+            "type": "object",
+            "properties": {
+               "EMDB_resolution": {
```

Comment:
Is it the same as "_em_3d_reconstruction.resolution"? If yes, we already load it

Reply:
Thank you @valasatava. Addressed this and the previous comment in the new commit.